Trimble.Modus.Components/Handlers/TMButtoniOSTouchHandler.cs

```diff
@@ -0,0 +1,50 @@
+﻿#if IOS
+using Foundation;
+using Microsoft.Maui.Controls.Handlers.Compatibility;
+using Trimble.Modus.Components.Controls.Button;
+using UIKit;
+
+namespace Trimble.Modus.Components.Handlers;
+
+public class TMButtoniOSTouchHandler : VisualElementRenderer<CustomButton>
+{
+    public TMButtoniOSTouchHandler()
+    {
+        if (Element.IsDisabled)
```

Comment:
The Element is null at the constructor. We have to use OnElementChanged override method to do this

Reply:
Modified the isdisabled handling by moving it on action occurrence